client/src/pages/Search/MapView/RestaurantPreview.js

```diff
@@ -1,7 +1,111 @@
-function RestaurantPreview({ restaurant }) {
+import React from "react";
+import ReviewStars from "../../../components/ReviewStars";
+import { IconButton } from "@mui/material";
+import CancelIcon from "@mui/icons-material/Cancel";
+import { Link } from "react-router-dom";
+
+const classes = {
+  previewContainer: {
+    height: "120px",
+    width: "100%",
+    padding: "0.25rem",
+    boxSizing: "border-box",
+    backgroundColor: "rgb(224,224,224)",
+    display: "flex",
+    justifyContent: "space-between",
+    border: "1px solid gray",
+    borderOffset: "-1px",
+    borderTop: "none",
+  },
+  imageContainer: {
+    height: "100%",
+    width: "120px",
+    borderRadius: "2px",
+    objectFit: "cover",
+    objectPosition: "center center",
+  },
+  detailsContainer: {
+    marginLeft: "0.25rem",
+    display: "flex",
+    flexDirection: "column",
+    fontSize: "0.8rem",
+  },
+  nameContainer: {
+    fontSize: "1.25rem",
+    textDecoration: "underline",
+    color: "black",
+    width: "fit-content",
+  },
+  categoriesContainer: {
+    marginTop: "auto",
+    color: "rgb(105,105,105)",
+  },
+  reviewContainer: {
+    fontSize: "0.75rem",
+    display: "flex",
+    color: "rgb(80,80,80)",
+  },
+  exitContainer: {
+    postion: "absolute",
+    top: 0,
+    right: 0,
+  },
+};
+
+function formatPhoneNumber(phoneNumberString) {
+  // Resource: https://stackoverflow.com/questions/8358084/regular-expression-to-reformat-a-us-phone-number-in-javascript
+  const cleaned = ("" + phoneNumberString).replace(/\D/g, "");
```

Comment:
why do we need an empty string here?

Reply:
unsure honestly, not my algo :( will fix it up tho